docs/getting-started/auth-self-defence-checks.adoc

```diff
@@ -28,26 +28,36 @@ There are two ways to implement self defence checks: <<Provided Self Defence Che
 
 These are self defence checks provided by the SDK in link:javadoc[`SecurityCheckType`].
 
-==== link:javadoc[IS_ROOTED]
+// TODO: make check titles links to javadoc when javadoc location is available
```

Comment:
Please don't have todo's in the docs.

Reply:
Cool, I will remove that now, I also created an issue for it here [#132](https://github.com/aerogear/aerogear-android-sdk/issues/132)